### PR TITLE
Feat/charts

### PR DESCRIPTION
**Pull request by maruf-ml**
Added a few charts as example. Tried passing props as well.

### CODE REVIEW

File: components/Header.vue [hunk 1]
```diff
@@ -5,7 +5,7 @@ const { logout } = useAuthStore();
 const { isLoggedIn } = storeToRefs(useAuthStore());
 
 const handleLogout = async () => {
-  logout();
+  await logout();
```
Comment (shimulH0) [inline components/Header.vue]: why await here?

File: components/PrivateHeader.vue [hunk 2]
```diff
@@ -13,48 +13,27 @@ function onChange(value: string) {
...
-          <li class="nav-item dropdown">
-            <select v-model="selectedValue" class="nav-link dropdown-toggle" @change="onChange(selectedValue)">
-              <option v-for="item in locales" :key="item.code" :value="item.code">
-                {{ item.name }}
-              </option>
-            </select>
-          </li>
-        </ul>
-      </div>
+    <nav>
```
Comment (shimulH0) [inline components/PrivateHeader.vue]: did you put and design again?? azmul bhai changed the project to bootstrap.

**Review by shimulH0 — CHANGES_REQUESTED** [review]

File: components/Header.vue (hunk 1 above)
Comment (maruf-ml) [reply components/Header.vue]: weird I don't even remember adding it :/ Removing it now.

File: components/PrivateHeader.vue (hunk 2 above)
Comment (maruf-ml) [reply components/PrivateHeader.vue]: thanks for noticing. due to the linter issue my files and changes were being removed so undoing caused this. Fixing it.

**Review by shimulH0 — APPROVED** [review]

**Pull request merged**